site/pages/gcweb-theme/static-header-footer/bootstrap-4.html

```diff
@@ -16,7 +16,7 @@
 	<!-- END - Experimental Bootstrap 4 -->
 
 </head>
-<body vocab="http://schema.org/" typeof="WebPage">
+<body vocab="http://schema.org/" resource="#wb-webpage" typeof="WebPage">
 
 <header class="header-b-line">
 	<div id="wb-bnr" class="container">
```

Comment:
Can you add the RDFa markup, like we have for the bootstrap 3 template, for the publisher area.
Here is line 35 to 40